controllers/operands/networkAddons.go

```diff
@@ -149,6 +150,12 @@ func NewNetworkAddons(hc *hcov1beta1.HyperConverged, opts ...string) (*networkad
 		if baseDomain != "" {
 			cnaoSpec.KubeSecondaryDNS.Domain = baseDomain
 		}
+		if nameServerIP := *hc.Spec.KubeSecondaryDNSNameServerIP; nameServerIP != "" {
+			if !net.IsIPv4String(nameServerIP) {
+				return nil, errors.New("kubeSecondaryDNSNameServerIP isn't a valid IPv4")
```

Comment:
are you sure the error will be shown to the user in realtime, or just CNAO CR won't be created and this will be shown only in the HCO logs?

Reply:
The ValidateCreate would fail